lib/instrumentation/http-shared.js

```diff
@@ -151,8 +153,11 @@ exports.traceOutgoingRequest = function (agent, moduleName, method) {
       // Use the transaction context as the parent, in this case.
       var parent = span || agent.currentTransaction
       if (parent && parent._context && shouldPropagateTraceContext(options)) {
-        const headerValue = parent._context.toString()
+        const headerValue = parent._context.toTraceParentString()
+        const traceStateValue = parent._context.toTraceStateString()
+
         options.headers.traceparent = headerValue
+        options.headers.tracestate = traceStateValue
         if (agent._conf.useElasticTraceparentHeader) {
```

Comment:
Here's where we _set_ the `tracestate` header on outgoing http transactions.  We are _not_ setting or giving an option to set the old `elastic-apm-tracestate` header in an attempt to keep things simple.  If a customer ends up needed this functionality we can add it later with relative simplicity.